db/database.go

```diff
@@ -79,6 +80,7 @@ type DatabaseContext struct {
 	State              uint32                  // The runtime state of the DB from a service perspective
 	ExitChanges        chan struct{}           // Active _changes feeds on the DB will close when this channel is closed
 	OIDCProviders      auth.OIDCProviderMap    // OIDC clients
+	PurgeInterval      int                     // Metadata purge interval, in hours
```

Comment:
Since this is only relevant for Couchbase buckets, should it mention that in the variable name (CouchbaseBucketPurgeInterval) or comment, or maybe be moved into the GoCouchbaseBucket struct?